src/main/java/Task_28/Elements.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Химические элементы</title>
+</head>
+<body>
+<p>
+    <b>Химические элементы</b>
+</p>
+<hr>
+
+<table>
```

Comment:
<table> tag should ideally contain <thead> and <tbody>

Reply:
Sorry, can't understand what you mean

Reply:
Sorry(
Git hurst my comment
`table` tag should ideally contain `thead` and `tbody`